build.gradle

```diff
@@ -78,6 +78,9 @@ subprojects {
             exclude project.property('excludeTests')
         }
 
+        // Allow running different test classes in the same project in parallel
+        maxParallelForks = Runtime.runtime.availableProcessors()
```

Comment:
I think it's probably fair to leave all 'regular' tests run in process.  That might net a marginal savings for the number of projects.  If it needs to be in its own process, it's likely a slow test anyway (forking & IPC will create a minor hit)

Reply:
This is because different tasks are not running in parallel, e.g. RFS and CoreUtilities will not run at the same time.

If i did that, i ran into an issue where having different project's slowTasks running at once could overload the docker memory allocation